image-customization.lua

```diff
@@ -0,0 +1,206 @@
+#!/usr/bin/lua
+
+features{
+    'autoupdater',
+    'config-mode-domain-select',
+    'config-mode-geo-location-osm',
+    'config-mode-mesh-vpn',
+    'ebtables-filter-multicast',
+    'ebtables-filter-ra-dhcp',
+    'ebtables-source-filter',
+    'mesh-batman-adv-15',
+    'respondd',
+    'status-page',
+    'web-advanced',
+    'web-private-wifi',
+    'web-wizard',
+}
+
+packages{
+    'ffac-autoupdater-wifi-fallback',
+    'ffac-change-autoupdater',
+    'ffac-ssid-changer',
+    'ffho-ap-timer',
+    'ffho-web-ap-timer',
+    'ffmuc-ipv6-ra-filter',
+    'ffmuc-mesh-vpn-wireguard-vxlan',
+    'iwinfo',
+    'respondd-module-airtime',
+}
+
+if not device_class('tiny') then
+    features{
+        'wireless-encryption-wpa3',
+    }
+end
+
+if device({
+        'zte,mf281',
+        'glinet,gl-xe300',
+        'glinet,gl-ap1300',
+        'zte,mf289f',
+        'wavlink,ws-wn572hp3-4g',
+        'tplink,tl-mr6400-v5',
+    }) then
+    features {
+        'web-cellular',
+    }
+end
+
+pkgs_usb = {
+    'usbutils',
+}
+
+pkgs_hid = {
+    'kmod-usb-hid',
+    'kmod-hid-generic',
+}
+
+pkgs_usb_serial = {
+    'kmod-usb-serial',
+    'kmod-usb-serial-ftdi',
+    'kmod-usb-serial-pl2303',
+}
+
+pkgs_usb_storage = {
+    'block-mount',
+    'blkid',
+    'kmod-fs-ext4',
+    'kmod-fs-ntfs',
+    'kmod-fs-vfat',
+    'kmod-usb-storage',
+    'kmod-usb-storage-extras',-- Card Readers
+    'kmod-usb-storage-uas', -- USB Attached SCSI (UAS/UASP)
+    'kmod-nls-base',
+    'kmod-nls-cp1250',      -- NLS Codepage 1250 (Eastern Europe)
+    'kmod-nls-cp437',       -- NLS Codepage 437 (United States, Canada)
+    'kmod-nls-cp850',       -- NLS Codepage 850 (Europe)
+    'kmod-nls-cp852',       -- NLS Codepage 852 (Europe)
+    'kmod-nls-iso8859-1',   -- NLS ISO 8859-1 (Latin 1)
+    'kmod-nls-iso8859-13',  -- NLS ISO 8859-13 (Latin 7; Baltic)
+    'kmod-nls-iso8859-15',  -- NLS ISO 8859-15 (Latin 9)
+    'kmod-nls-iso8859-2',   -- NLS ISO 8859-2 (Latin 2)
+    'kmod-nls-utf8',        -- NLS UTF-8
+}
+
+pkgs_usb_net = {
+    'kmod-mii',
+    'kmod-usb-net',
+    'kmod-usb-net-asix',
+    'kmod-usb-net-asix-ax88179',
+    'kmod-usb-net-cdc-eem',
+    'kmod-usb-net-cdc-ether',
+    'kmod-usb-net-cdc-subset',
+    'kmod-usb-net-dm9601-ether',
+    'kmod-usb-net-hso',
+    'kmod-usb-net-ipheth',
+    'kmod-usb-net-mcs7830',
+    'kmod-usb-net-pegasus',
+    'kmod-usb-net-rndis',
+    'kmod-usb-net-rtl8152',
+    'kmod-usb-net-smsc95xx',
+}
+
+pkgs_pci = {
+    'pciutils',
+    'kmod-bnx2', -- Broadcom NetExtreme BCM5706/5708/5709/5716
+}
+
+include_tls = not device({
+    'd-link-dir825b1',
```

Comment:
The dir825b1 is also part of `tiny` - so we could move TLS to if not tiny?

Reply:
@maurerle from what I can tell, all other devices, except dir825b1 build fine with TLS even if they are tiny.